pkg/network/protocols/kafka/testdata/docker-compose.yml

```diff
@@ -9,16 +8,16 @@ services:
       - ALLOW_ANONYMOUS_LOGIN=yes
     tmpfs:
       - /bitnami/zookeeper/data
-  kafka:
+  kafka-old:
     image: bitnami/kafka:3.9
     ports:
-      - "${KAFKA_PORT:-9092}:9092"
-      - "${KAFKA_TLS_PORT:-9093}:9093"
+      - "9082:9082"
+      - "9083:9083"
     environment:
       - KAFKA_ENABLE_KRAFT=false
       - KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=CLIENT:PLAINTEXT,SSL:SSL
-      - KAFKA_CFG_LISTENERS=CLIENT://:9092,SSL://:9093
-      - KAFKA_CFG_ADVERTISED_LISTENERS=CLIENT://${KAFKA_ADDR:-127.0.0.1}:${KAFKA_PORT:-9092},SSL://${KAFKA_ADDR:-127.0.0.1}:${KAFKA_TLS_PORT:-9093}
+      - KAFKA_CFG_LISTENERS=CLIENT://:9082,SSL://:9083
+      - KAFKA_CFG_ADVERTISED_LISTENERS=CLIENT://${KAFKA_ADDR:-127.0.0.1}:9082,SSL://${KAFKA_ADDR:-127.0.0.1}:9083
```

Comment:
why do we need old and new?

Reply:
Kafka 4.0 dropped support for old versions of the protocol :(

Reply:
1. Then let's document it in the docker compose
2. Let's create a ticket in our board to iterate over the kafka tests, and allowing only one container to start (as running docker is time consuming, we should try to have the minimal setup)

Reply:
Also, it should be highlighted in the PR description for clarifty

Reply:
https://datadoghq.atlassian.net/browse/USMON-1436